### ISSUE TITLE
LMDB Dataset & Loss Function Calculation

### ISSUE THREAD

**Issue by ragilhadi3** [wp03052/MATRN#20]
Hey there,

I've got a couple of questions I'd like to throw your way:

- So, I've been tinkering around with MATRN and trying to train it on my own custom dataset using PyTorch Lightning. I've been trying to mimic your approach with fastai, but it seems like my model isn't learning anything. I've got a feeling it's something to do with how I'm calculating the loss using multiloss. Even though I borrowed some code from yours for the multiloss, I'm still hitting the same snag. Any chance you could help me figure out the right way to calculate the loss during training?
- Also, I'm hitting a bit of a roadblock while trying to convert my own dataset into an LMDB dataset for training using your code. Any tips on how the input path structure should look when converting it into an LMDB dataset?

Thanks a bunch for your help,  your MATRN architecture is wicked cool!
